test_fms/mpp/Makefile.am

```diff
@@ -50,21 +51,24 @@ test_mpp_print_memuse_stats_stderr_SOURCES=test_mpp_print_memuse_stats_stderr.F9
 test_mpp_print_memuse_stats_file_SOURCES=test_mpp_print_memuse_stats_file.F90
 test_mpp_memutils_begin_2x_SOURCES=test_mpp_memutils_begin_2x.F90
 test_mpp_memutils_end_before_begin_SOURCES=test_mpp_memutils_end_before_begin.F90
+test_system_clock_SOURCES=test_system_clock.F90
 
 # Run the test programs.
 TESTS = test_mpp_domains2.sh \
   test_mpp_pset2.sh \
   test_mpp2.sh \
   test_mpp_memuse \
   test_mpp_mem_dump \
-  test_mpp_memutils_mod.sh
+  test_mpp_memutils_mod.sh \
+  test_system_clock.sh
 
 # These files will also be included in the distribution.
 EXTRA_DIST = input_base.nml \
   test_mpp_domains2.sh \
   test_mpp_pset2.sh	\
   test_mpp2.sh \
-  test_mpp_memutils_mod.sh
+  test_mpp_memutils_mod.sh \
+  test_system_clock.sh
 
 # Clean up
 CLEANFILES = input.nml *.out*
```

Comment:
Add your created module (include_files_mod.mod) here to clean it up.

Reply:
Thank you - that worked and Travis CI tests pass now.